include/dts-functions.sh

```diff
@@ -1449,9 +1449,12 @@ main_menu_options(){
       return 0
       ;;
     "${DPP_KEYS_OPT}")
-      local _result
```

Comment:
Any reason you have deleted declaration of this variable?

Reply:
Oops, bug from rebasing on main

Reply:
a79f9a1c4328b3ea748b73c807f5f68a7a18a321

Reply:
You should not change anything unless you have a reason. 2561922042e60f3fbcd2aae75982a186fd245eae.